firebase-vertexai/src/main/kotlin/com/google/firebase/vertexai/type/LiveSession.kt

```diff
@@ -310,7 +310,7 @@ internal constructor(
   ) {
     receive()
       .transform {
-        if (it.status == LiveContentResponse.Status.INTERRUPTED) {
+        if (it.interrupted == true) {
```

Comment:
```suggestion
        if (it.interrupted) {
```

Reply:
Unfortunately, this doesn't work since it's nullable.